web-app/templates/book/createBookView.html

```diff
@@ -0,0 +1,80 @@
+<form name="createBookForm" ng-submit="createNewBook()">
```

Comment:
If you use controllerAs syntax, Try to avoid $scope methods and properties.

Check everywhere.

Reply:
Move templates to relative folder "book" and rename to create-book.(tmpl | view | template).html